remill/Arch/AArch64/Extract.cpp

```diff
@@ -47970,10 +47970,17 @@ static bool TryExtractCINC_CSINC_32_CONDSEL(InstData &inst, uint32_t bits) {
   inst.cond = static_cast<uint8_t>(enc.cond);
   if (!(inst.cond != 0xe)) return false;
   if (!(inst.cond != 0xf)) return false;
+  // Condition has to be inverted for the alias to work
+  // when converted to csinc instruction so we toggle last bit
+  inst.cond ^= 1;
```

Comment:
I don't understand this. This seems like a "destructive" change, where this inversion should really be processed inside of the `TryDecode` function. Is inverting here necessary to handle aliasing?